lang/en.json

```diff
@@ -225,6 +225,7 @@
     },
     "category": {
         "label": "Categories",
+        "product_label": "Product filter",
```

Comment:
@bc-vlad-dlogush also my suggestion is to change the key value to:
`"Product list filter"` or `"Products filter"`